Makefile

```diff
@@ -48,7 +48,7 @@ push-image:
 
 test: codegen download-test-deps
 	KUBEBUILDER_ASSETS="$(shell setup-envtest --use-env use -p path)" \
-		ginkgo -nodes=${TEST_NODES} --randomize-all ./test/...
+		ginkgo -nodes=${TEST_NODES} --randomizeAllSpecs ./test/...
```

Comment:
Earlier in the branch I changed all the ginkgo imports to v2; this flag was respelled in the command line v2 (it'll accept the old spelling, but complain).

Reply:
Looks like it'll all work with ginkgo CLI v1, so it can be left where it is. I don't know when I got the idea that the command line needed to be updated too. (but you certainly can't mix the versions in imports, they define the same flags which leads to a panic)

Reply:
I think the GitHub Action code must still be using the old CLI then. The test was failing with an unknown option error.

Reply:
I'll see if I can update the CLI while I'm at it

Reply:
Yeah, I'd changed that locally then was surprised by new commits on the branch :grin: If it works with the old one, not changing it saves people downloading it. So I'm happy to just revert the flag name.